integration-tests/test/lib/JavabuilderConnectionHelper.js

```diff
@@ -0,0 +1,99 @@
+import { v4 as uuidv4 } from "uuid";
+import fetch from "node-fetch";
+import jwt from "jsonwebtoken";
+import crypto from "crypto";
+import WebSocket from "ws";
+import {
+  JAVABUILDER_HTTP_URL,
+  JAVABUILDER_WEBSOCKET_URL,
+  PASSWORD,
+  PRIVATE_KEY,
+} from "./environment.js";
+
+// TODO: use integration test specific origin once the key pair has been setup.
```

Comment:
do we have a task for this already?

Reply:
Yep! https://codedotorg.atlassian.net/browse/JAVA-588